public/app/plugins/datasource/prometheus/querybuilder/components/MetricSelect.tsx

```diff
@@ -57,29 +61,32 @@ export function MetricSelect({
     },
   ];
 
-  const customFilterOption = useCallback((option: SelectableValue<any>, searchQuery: string) => {
-    const label = option.label ?? option.value;
-    if (!label) {
-      return false;
-    }
+  const customFilterOption = useCallback(
+    (option: SelectableValue<any>, searchQuery: string) => {
```

Comment:
Sorry this is my OCD :) Do we need that indentation?

Reply:
I tried to remove the indentation but prettier added it back. Placing the function argument inline with the useCallback function makes the line too long. Therefore, we get the indent 🤷🏻